### PR TITLE
Drop unused python2-qpid-qmf

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: manifests/params.pp [hunk 1]
```diff
@@ -32,7 +32,7 @@
   $user = 'qpidd'
   $group = 'qpidd'
 
-  $server_packages = ['qpid-cpp-server', 'qpid-cpp-client', 'python2-qpid-qmf', 'python2-qpid', ]
+  $server_packages = ['qpid-cpp-server', 'qpid-cpp-client', 'python2-qpid', ]
```
Comment (ekohl) [inline manifests/params.pp]:
```suggestion
  $server_packages = ['qpid-cpp-server', 'qpid-cpp-client', 'python2-qpid']
```

**Comment by ehelms** [comment]
Yea, `python2-qpid` is still needed from everything I see.

**Comment by ehelms** [comment]
@ekohl Can you provide me an example commit that uses github issue like you mean? Not seeing that format in the autoclose list for github.

**Comment by ekohl** [comment]
https://github.com/theforeman/puppet-puppet/commit/27eb207440f450b7cfdbd678100b79e91031f4ce is a format we commonly use

**Pull request merged**